airflow/dag_processing/manager.py

```diff
@@ -1022,8 +1012,8 @@ def start_new_processes(self):
                 continue
 
             callback_to_execute_for_file = self._callback_to_execute[file_path]
-            processor = self._processor_factory(
-                file_path, callback_to_execute_for_file, self._dag_ids, self._pickle_dags
+            processor = type(self)._create_process(
```

Comment:
```suggestion
            processor = self._create_process(
```

Reply:
It's actually a static method but I'm still considering another option here though.

Reply:
`type(self)._create_process` is a "trick" to avoid multiprocessing pickling more than it should (taken from the MP source code), which was important when the factory method was being sent from the Scheduler to the DagFileProcessorManager over a multiprocessing connection

But since we are now just creating the processor, rather than passing a reference to the the factory, we can do as Jed suggests.

And `self._some_static_method` works fine.

Reply:
Awesome! It got me thinking it’s the only way to use static method in its class. Thanks for this!